snapshots/src/unarchive.rs

```diff
@@ -52,12 +52,6 @@ fn decompressed_tar_reader(
     buf_size: u64,
 ) -> io::Result<ArchiveFormatDecompressor<impl BufRead>> {
     let buf_reader =
-        solana_accounts_db::large_file_buf_reader(archive_path.as_ref(), buf_size as usize)
-            .map_err(|err| {
-                io::Error::other(format!(
-                    "failed to open snapshot archive '{}': {err}",
-                    archive_path.as_ref().display(),
-                ))
```

Comment:
I see we put the archive's path in the error here. Why is it not logged currently?

Reply:
the error will be something like
`Unpacking '/some/path.tar.zst' failed: IO error: invalid frame`

Reply:
we will not distinguish error from opening from error during processing, we will report what was the input path and what was the underlying error raised by inner component